net.brinkervii.grapejuice.yml

```diff
@@ -234,8 +234,8 @@ modules:
       - cp /app/share/icons/hicolor/scalable/apps/grapejuice.svg /app/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.png
     sources:
       - type: archive
-        url: https://gitlab.com/brinkervii/grapejuice/-/archive/v7.19.8/grapejuice-v7.19.8.tar.bz2
-        sha256: 07e4c5b6cc02dc9d3a8dc5c0069154ca3dfa91f29666a35a7bfed8524097ad16
+        url: https://gitlab.com/brinkervii/grapejuice/-/archive/v7.14.4/grapejuice-v7.19.8.tar.bz2
```

Comment:
Did you mean to downgrade Grapejuice itself as well?

Reply:
Yes, apparently Brinker's new env sanitization thingy is not working properly

Reply:
Ignore what I said, looks like Brinker has a brand new release already.